test/src/core/testPageObjectWaitForElementNotPresent.js

```diff
@@ -18,16 +18,17 @@ describe('test PageObject WaitForElementNotPresent', function () {
     Nocks.deleteSession().disable();
   });
 
-  it('WaitForElementNotPresent with section', function(done) {
+  it('WaitForElementNotPresent with section', async function() {
     Nocks.elementFound().click().elementFound().childElementsFound('#badElement').elementFound().childElementsNotFound()
 
     const page = this.client.api.page.waitForElementNotPresentPageObj();
 
+    let res;
     page.waitForElementNotPresentDemo(function(result) {
-      assert.equal(result.status, 0);
-      done();
+      res = result;
     })
-
-    this.client.start()
+  
+    await this.client.start();
```

Comment:
does this guarantee that the callback to`waitForElementNotPresentDemo` has executed?

Reply:
the `await` returns when `queue.run()` finishes, that means all the callbacks have been executed right?

Reply:
I am pretty sure it does